deploy/operator.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     spec:
       containers:
         - name: keycloak-operator
-          image: quay.io/integreatly/keycloak-operator:v1.2.1
+          image: quay.io/integreatly/keycloak-operator:1.2.1
```

Comment:
`quay.io/integreatly/keycloak-operator:1.2.1` is a manual build, is that correct? Is the plan to move this back to a build from a tag eventually?